bin/kalite

```diff
@@ -58,6 +58,10 @@ else:
     if kalite_path and not os.path.exists(os.path.join(kalite_path, 'kalite', '__init__.py')):
         kalite_path = None
 
+    installed_path = os.path.join(sys.prefix, 'share', 'kalite')
```

Comment:
No, unfortunately not atm... the idea is that bin/kalite never imports kalite itself... it just reads some environment and opens up a subprocess invoking the correct python VM and with some KALITE_DIR env setting.